### ISSUE TITLE
Multi calendar with solo enabled jumps to different dates after first selection

### ISSUE THREAD

**Issue by RiesvGeffen** [Vuepic/vue-datepicker#812]
**Describe the bug**
After selecting the first date (so range is enabeld) on the right calendar with `multi-calendars` it's `solo` property on `true`. The calendar on the right jumps to the connecting month of the left. It's the same issue as the last one I reported #649, that might help with finding the fix.

**To Reproduce**
1. Go to https://stackblitz.com/edit/vuepic-vue-datepicker-duh3f6?file=package.json
2. Open the datepicker
3. For example, click on 2023 November 1

**Expected behavior**
Calendar on the right should stay on 2023 November(, but it jumps to 2021 December). I would expect both sides to stay on the month and year what they where on.

**Desktop & mobile:**

- Chrome: 123.0.6312.107
- Library version: 8.4.0

**Comment by Jasenkoo** [comment]
This won't be altered as it is expected behavior, these offset dates should not represent the selected state if another month where the date is selected is visible.

**Comment by RiesvGeffen** [comment]
So in #649 I filed a bug report for this and it was fixed. In this current issue I filed the exact same bug report and it was communicated with me that it won't be fixed. Now I check the newest version, maybe it was fixed some version ago already, but now it seems to be fixed.

What was communicated with me, that the other behavior was as expected, does not seem logical to me. What is the purpose of a multi solo calendar, if selecting something on one side, makes the other side jump to the same side. That makes it completely useless.

Just wanted to come back to this bug report, to ask for you in the future to re-consider some decisions, before just closing a bug report.

I, nonetheless, greatly appreciate all you do!
